agent/api/task.go

```diff
@@ -72,11 +72,11 @@ type Task struct {
 	Containers []*Container
 	// Volumes are the volumes for the task
 	Volumes []TaskVolume `json:"volumes"`
-	// VCPULimit is a task-level limit for compute resources. A value of 1 means that
+	// Cpu is a task-level limit for compute resources. A value of 1 means that
 	// the task may access 100% of 1 vCPU on the instance
-	VCPULimit float64 `json:"CPULimit,omitempty"`
-	// MemoryLimit is a task-level limit for memory resources in bytes
-	MemoryLimit int64 `json:"Memory,omitempty"`
+	Cpu float64 `json:"CPULimit,omitempty"`
+	// Memory is a task-level limit for memory resources in bytes
+	Memory int64 `json:"Memory,omitempty"`
```

Comment:
We aren't plumbing the ACS model data yet?

Reply:
The plumbing should be reflection based unless I'm mistaken